### PR TITLE
VM images: bump to 2023-11-16

### PR DESCRIPTION
**Pull request by edsantiago**
No change in base, just newer packages (especially crun-1.11-1)

Then fix broken tests.

Signed-off-by: Ed Santiago <santiago@redhat.com>
```release-note
None
```

### CODE REVIEW

**Comment by edsantiago** [comment]
All sorts of failures with `zstd` in the message. @flouthoc @rhatdan PTAL. This is a VM-only change, not a real PR.

**Comment by rhatdan** [comment]
@giuseppe PTAL

**Comment by flouthoc** [comment]
Test was added by me, but I don't know in which PR the default compression is changed to `zstd-chunked`, tests expects default compression to be `gzip`. Is it changed in the VM itself ? otherwise this should have failed in the PR where this change was made.  @giuseppe Any ideas if default compression is changed to `zstd-chunked` for all compression ?

**Comment by martinpitt** [comment]
@flouthoc : that change was introduced in containers-common, via direct upload to Fedora/podman-next COPR, not via a PR. See #20611

**Comment by flouthoc** [comment]
ah it is that rawhide now defaults to `zstd-chunked`

**Comment by rhatdan** [comment]
Yes someone has to fix the tests or figure out if zstd:chunked is broken.  I am at KubeCon and have no time to investigate.

**Comment by flouthoc** [comment]
@rhatdan @edsantiago Created a PR https://github.com/containers/podman/pull/20617

**Comment by vrothberg** [comment]
@rhatdan can we revert please?  This is blocking feature work in https://github.com/containers/podman/pull/20356.

**Comment by edsantiago** [comment]
Executive decision, I am forcing an override in the CI images. New VM build in progress.

**Comment by rhatdan** [comment]
Reverting this is just pushing it down the road. We need to figure these issues out and get tests to pass.

**Comment by edsantiago** [comment]
Full consensus and agreement from the team (we discussed this in scrum). We just decided to get the problems ironed out first, and _then_ change the default.

**Comment by vrothberg** [comment]
> Reverting this is just pushing it down the road. We need to figure these issues out and get tests to pass.

Yes, tests need to pass.  But reverting is inevitable in my eyes since we can't just wait for somebody to do something at some point.  Who's responsible for fixing it?

I think we assumed that tests will pass without actually testing it.  There might be real issues that require more than massaging tests.

**Comment by giuseppe** [comment]
tests pass with https://github.com/containers/podman/pull/20633

We need the famous fix for c/image as well as another one for blobcache.  Only a few tests need to be adjusted as they assume gzip or the docker format is used.

I think the safest is to revert the change in Rawhide since it affects users and we know there are real issues at the moment (like the PR for c/image).  Once we settle on that, we need to change the default in c/common and seal our CI with it.

**Comment by edsantiago** [comment]
@cevich, @containers/podman-maintainers PTAL. This is a bump to release f39 (up from beta), and to crun-1.11.2 on all Fedoras (debian uses runc so we don't care about crun). See [package version list](https://github.com/containers/automation_images/pull/312#issuecomment-1815338192).

File: test/e2e/run_test.go [hunk 1]
```diff
@@ -1545,14 +1545,18 @@ VOLUME %s`, ALPINE, volPath, volPath)
 		container.WaitWithDefaultTimeout()
 		Expect(container).Should(Exit(0))
 		checkLines(container.OutputToStringArray())
-		Expect(container.ErrorToString()).To(ContainSubstring("Running scope as unit: "))
```
Comment (edsantiago) [inline test/e2e/run_test.go:1548]: https://github.com/systemd/systemd/commit/7693661a8406e004dc20b696850c9acb8c024566

**Review by Luap99 — APPROVED** [review]
LGTM

**Pull request merged**